engines/base/src/main/resources/templates/test.html

```diff
@@ -0,0 +1,58 @@
+<html xmlns:th="http://www.thymeleaf.org">
+<body>
+	<div>
+        <h2 th:text="${title}"></h2>
+        <form method="POST" enctype="multipart/form-data" th:action="${proxyPathPrefix+'/test'}">
+            <table>
```

Comment:
Optional:
There are no `<th>` in this `<table>`. I understand if this is for testing so feel free to ignore.

Reply:
Yes for testing. Not adding the pretty stuff in the headers.